migrations/env.py

```diff
@@ -0,0 +1,70 @@
+from alembic import context
+from sqlalchemy import engine_from_config, pool
+from webapp.models import Base
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+target_metadata = Base.metadata
+
+# other values from the config, defined by the needs of env.py,
+# can be acquired:
+# my_important_option = config.get_main_option("my_important_option")
+# ... etc.
+
+
+def run_migrations_offline():
```

Comment:
I'm gonna need to look into this `offline` and `online` migrations concept more deeply. At the moment, it confuses me.

Reply:
https://alembic.sqlalchemy.org/en/latest/offline.html

Reply:
I don't think we need offline migrations do we? They make me nervous. We should always have a reliable database to test the import against.

Reply:
The offline mode is used when you use flags like `--sql` and other commands that might not need the actual connection to the database.